netlify.toml

```diff
@@ -1,4 +0,0 @@
-[build]
-  publish = "frontend/build/"
-  functions = "backend/build"
-  command = "npm run build"
```

Comment:
Is there a reason why it's removed?

Reply:
this is removed because the dti site needed a different publish directory and it was overriding the one I put in the netlify website lol